rmf_traffic/src/rmf_traffic/schedule/Negotiation.cpp

```diff
@@ -455,7 +455,9 @@ class Negotiation::Table::Implementation
     for (const auto& p : proposal)
     {
       const ParticipantId participant = p.participant;
-      const auto& description = schedule_viewer->get_participant(participant);
+      const auto& description =
```

Comment:
This should technically be okay because "The lifetime of a temporary object may be extended by binding to a const lvalue reference" [[1]](https://en.cppreference.com/w/cpp/language/lifetime#Temporary_object_lifetime), but I think it's better to make this a true lvalue instead of a const lvalue reference so we're not playing games with the convoluted rules of lifetimes in C++.

```suggestion
      const auto description =
```

Reply:
This is probably a typo because I got too used to typing `const auto&` in for loops. :)